sdk/core/src/CommonNetworkMember/LegacyNetworkMemberWithFactories.ts

```diff
@@ -6,13 +6,20 @@ import { withDidData } from '../shared/getDidData'
 import { getOptionsFromEnvironment, ParsedOptions } from '../shared/getOptionsFromEnvironment'
 import { ParametersValidator } from '../shared/ParametersValidator'
 import { randomBytes } from '../shared/randomBytes'
-import { validateUsername } from '../shared/validateUsername'
-import UserManagementService from '../services/UserManagementService'
+import { validateUsername, UserManagementService } from '@affinidi/user-management'
 import { StaticDependencies, ConstructorUserData, createKeyManagementService } from './BaseNetworkMember'
 import { LegacyNetworkMember } from './LegacyNetworkMember'
+import { DEFAULT_COGNITO_REGION } from '../_defaultConfig'
+import { KeyStorageApiService } from '@affinidi/internal-api-clients'
```

Comment:
Please maintain the alphabetic order of dependencies, in this line and in line 9.

(We really should update eslint-config dependency, the newer config should catch things like that)